src/index.jsx

```diff
@@ -37,7 +38,7 @@ class CircularProgressbar extends React.Component {
   }
 
   getPathDescription() {
-    const radius = this.getRadius();
+    const radius = this.getRadius() - BACKGROUND_OFFSET;
```

Comment:
this change causes a discrepancy with regard to the path's radius and the diameter of the path used in getProgressStyle, resulting in the path being too long for any given percentage:

<img width="100" alt="screen shot 2017-10-13 at 8 36 35 pm" src="https://user-images.githubusercontent.com/396006/31572823-70f78db2-b063-11e7-8de8-7b1343115de0.png">

there was no test case for it, so this is an understandable bug - fixed this and added test in https://github.com/iqnivek/react-circular-progressbar/pull/23/commits/d24cbe5a8167c9752d0135aafb7e73a4179c0674